NAMESPACE

```diff
@@ -1,19 +1,38 @@
 useDynLib(GeoDiff, .registration=TRUE)
 importFrom(Rcpp, evalCpp)
+importFrom(Rcpp,sourceCpp)
 
 
 import(Biobase)
```

Comment:
we probably need either `import(Biobase)` or all the `importFrom(Biobase,...)` lines, not sure which one is better

Reply:
> we probably need either `import(Biobase)` or all the `importFrom(Biobase,...)` lines, not sure which one is better

I removed the others.